setup.bat

```diff
@@ -0,0 +1,24 @@
+@echo off
+set "zipUrl=https://github.com/heysagnik/Linkees/archive/refs/heads/old.zip"

```

Comment:
How will the user receive updated zip files?

Reply:
Would you like it to be the master branch?